include/CoreGraphics/CGPath.h

```diff
@@ -19,7 +19,14 @@
 #import <CoreGraphics/CGGeometry.h>
 #import <CoreGraphics/CoreGraphicsExport.h>
 
-typedef enum { kCGPathFill, kCGPathEOFill, kCGPathStroke, kCGPathFillStroke, kCGPathEOFillStroke } CGPathDrawingMode;
+typedef enum {
+//  CG Drawing Bitfield   |STROKE|   |FILL  |   |EO    |
+    kCGPathStroke =       (1 << 0),
+    kCGPathFill =                    (1 << 1),
+    kCGPathFillStroke =   (1 << 0) | (1 << 1),
+    kCGPathEOFill =                  (1 << 1) | (1 << 2),
+    kCGPathEOFillStroke = (1 << 0) | (1 << 1) | (1 << 2),
+} CGPathDrawingMode;
```

Comment:
Didn't format it; I'll add some crazy format lines to keep this forever :)

---
In reply to: [83315342](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83315342) [](ancestors = 83315342)